apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformer.java

```diff
@@ -37,21 +43,61 @@ public MessageOrBuilder transform(Parser.Data<String> data) {
     if (isFirstLineNum(data)) {
       FissClaim.Builder claimBuilder =
           FissClaim.newBuilder()
-              // Prefixed DCN numbers with '*' to designate synthetic data
-              .setDcn(convertDcn(data))
+              .setDcn(
+                  // TODO: Do we need a switch?
```

Comment:
No we don't need a switch/flag for this; if the value is present use it otherwise generate a new DCN based on the CLM_ID value